dockers/app/requirements.pip3

```diff
@@ -1,9 +1,67 @@
 APScheduler==3.3.1
+asn1crypto==1.3.0
+attrs==19.3.0
+backcall==0.1.0
+bcrypt==3.1.7
+certifi==2019.11.28
+cffi==1.14.0
+chardet==3.0.4
+Click==7.0
+cryptography==2.8
+decorator==4.4.1
+Flask==0.12.5
+Flask-Login==0.5.0
 Flask-SocketIO==2.8.6
-iniparse==0.4
-rethinkdb==2.3.0.post6
+###Flask-SocketIO==4.2.1
```

Comment:
This comments could be deleted, since they're not used for anything

Reply:
ok